report_thesis/src/sections/baseline.tex

```diff
@@ -0,0 +1,52 @@
+In \citet{p9_paper}, we presented our efforts to replicate the MOC model by \citet{cleggRecalibrationMarsScience2017}.
+This effort was motivated by our desire to understand the model and its performance better, and to experiment with its components to determine how it could be improved.
```

Comment:
```suggestion
This effort was motivated by our desire to better understand the model and its performance, and to experiment with its components in order to determine how it could be improved.
```